index.js

```diff
@@ -1,176 +1,41 @@
-
-    while (permOrder.length) {
-      const currentLevel = permOrder.shift();
-      if (message.guild && currentLevel.guildOnly) continue;
-      if (currentLevel.check(message)) {
-        permlvl = currentLevel.level;
-        break;
-      }
-    }
-    return permlvl;
-  }
-
-  loadCommand(commandPath, commandName) {
-    try {
-      const props = new (require(`${commandPath}${path.sep}${commandName}`))(client);
-      // client.logger.log(`Loading Command: ${props.help.name}. 👌`, "log");
-      props.conf.location = commandPath;
-      if (props.init) {
-        props.init(client);
-      }
-      client.commands.set(props.help.name, props);
-      props.conf.aliases.forEach(alias => {
-        client.aliases.set(alias, props.help.name);
-      });
-      return false;
-    } catch (e) {
-      return `Unable to load command ${commandName}: ${e}`;
-    }
-  }
-
-  async unloadCommand(commandPath, commandName) {
-    let command;
-    if (client.commands.has(commandName)) {
-      command = client.commands.get(commandName);
-    } else if (client.aliases.has(commandName)) {
-      command = client.commands.get(client.aliases.get(commandName));
-    }
-    if (!command) return `The command \`${commandName}\` doesn"t seem to exist, nor is it an alias. Try again!`;
-
-    if (command.shutdown) {
-      await command.shutdown(client);
-    }
-    delete require.cache[require.resolve(`${commandPath}${path.sep}${commandName}.js`)];
-    return false;
-  }
-
-  getSettings(id) {
-    const defaults = client.settings.get("default");
-    let guild = client.settings.get(id);
-    if (typeof guild != "object") guild = {};
-    const returnObject = {};
-    Object.keys(defaults).forEach((key) => {
-      returnObject[key] = guild[key] ? guild[key] : defaults[key];
-    });
-    return returnObject;
-  }
-
-  writeSettings(id, newSettings) {
-    const defaults = client.settings.get("default");
-    let settings = client.settings.get(id);
-    if (typeof settings != "object") settings = {};
-    for (const key in newSettings) {
-      if (defaults[key] !== newSettings[key]) {
-        settings[key] = newSettings[key];
-      } else {
-        delete settings[key];
-      }
-    }
-    client.settings.set(id, settings);
-  }
-}
-
-const client = new Misaki({
-  fetchAllMembers: false,
+require(`${process.cwd()}/extenders/Message.js`);
+require(`${process.cwd()}/extenders/GuildMember.js`);
+require(`${process.cwd()}/extenders/Guild.js`);
+require(`${process.cwd()}/extenders/DMChannel.js`);
+require(`${process.cwd()}/extenders/TextChannel.js`);
+const MisakiClient = require(`${process.cwd()}/structures/MisakiClient.js`);
+
+const client = new MisakiClient({
+  disabledEvents: [
+    "CHANNEL_PINS_UPDATE",
+    "GUILD_BAN_ADD",
+    "GUILD_BAN_REMOVE",
+    "GUILD_SYNC",
+    "RELATIONSHIP_ADD",
+    "RELATIONSHIP_REMOVE",
+    "TYPING_START",
+    "USER_NOTE_UPDATE",
+    "USER_SETTINGS_UPDATE",
+    "VOICE_SERVER_UPDATE",
+    "VOICE_STATE_UPDATE"],
```

Comment:
I dislike how these are, pls 1 line them

Reply:
You cant read them when they're on one line.